src/many-ledger/src/storage.rs

```diff
@@ -181,6 +230,12 @@ impl LedgerStorage {
 
         self.latest_tid = TransactionId::from(height << HEIGHT_TXID_SHIFT);
 
+        if height % 1000 == 0 {
+            if let Err(e) = self.create_snapshot(height) {
+                println!("{}", e);
```

Comment:
Use `tracing::error!()` instead of println. Tracing has a lot of configurability so it's preferable.